### ISSUE TITLE
Document REFPROP version

### ISSUE THREAD

**Issue by jowr** [CoolProp/REFPROP-headers#4]
Could we please find a way to document which version of REFPROP is intended to be used with this repository? People with old REFPROP versions might find it useful to either have tags or branches for old REFPROP versions. We could also include some more macros to conditionally define the new functions, but I am afraid that this makes the code even harder to read.

**Comment by ibell** [comment]
Yeah this is a good idea.  I am actually quite lost with what version of REFPROP works with what version of CoolProp.  I have heard that CP 5.0.8 works with all versions of REFPROP, but newer versions don't.  I'd love to figure out why.
